spec/acceptance/rest/auth_spec.rb

```diff
@@ -1338,5 +1338,31 @@ def coerce_if_time_value(field_name, value, params = {})
         expect(response).to be_a(Ably::Models::TokenDetails)
       end
     end
+
+    context 'when using JWT' do
+      let(:auth_url) { 'https://shrouded-plains-50367.herokuapp.com/createJWT' } # TODO: change this
+      let(:token) { Faraday.get "#{auth_url}?keyName=#{key_name}&keySecret=#{key_secret}" }
```

Comment:
This is returning an HTTP response object, not a token

Reply:
Same as above

Reply:
See https://github.com/ably/ably-ruby/commit/ffd1e18a2bcc309fe2320c9d5a6fc54f4af8c464